softdev.bib

```diff
@@ -3333,3 +3333,10 @@ @article{yanovski21ghostcell
   year={2021},
   month = Aug,
 }
+
+@misc{cheng20typefreezing,
+    title = "Type freezing: exploiting attribute type monomorphism in tracing JIT compilers",
+    author = "Cheng, Lin and Ilbeyi, Berkin and Bolz-Tereick, Carl Friedrich and Batten, Christopher",
+    year = 2020,
+    isbn = {9781450370479},
```

Comment:
I that sufficient? or does it need more info?

Reply:
What event was this published at? Please see other entries in the file to get a sense of the possibilities.

Reply:
Is the event a publisher?
Added publisher :point_right: 98eb9ccdd96826c197611bd86e96311a75c06d25

Reply:
The publisher isn't relevant to us. This was published at a conference or workshop, probably? If you look at other entries you'll see how to deal with such a case.

Reply:
Looking at the other entries, I guess it should be set as `booktitle`, I didn't realise that it's used for events.
Updated :point_right: 2d59f6f340598a8bc335ff2f764c648e694a45b6

Reply:
That's reasonable but note you now need to make the entry `inproceedings`.

Reply:
ok. done :point_right: 118d3fc7fad24b55cc5aadaacd0ddce7a2408f6e